frontend/config-overrides.js

```diff
@@ -0,0 +1,16 @@
+// Config overrides for CRA to allow importing of aliases and the /shared folder.
+const {
+  aliasDangerous,
+  configPaths,
+  aliasJest,
+} = require('react-app-rewire-alias/lib/aliasDangerous')
+
+const aliasMap = configPaths('tsconfig.paths.json')
+
+module.exports = function override(config) {
+  aliasDangerous(aliasMap)(config)
```

Comment:
This is for aliasing outside of root folder; i.e having a `/shared` folder that may be outside of the react app's `/src` folder. Since that is not the case yet, will go back to using `alias`. Will add a comment